File: OCH_Tao/ABC/B/028.py
S = input()
l = []
l.append(S.count("A"))
l.append(S.count("B"))
l.append(S.count("C"))
l.append(S.count("D"))
l.append(S.count("E"))
l.append(S.count("F"))
print(" ".join(str(i) for i in l))

File: OCH_Tao/ABC/B/028.js
const S = (require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n"))[0];
let l = [];
const x = [/A/g, /B/g, /C/g, /D/g, /E/g, /F/g];
for (let i = 0; i < 6; i++) {
  let tmp = S.matchAll(x[i]);
  l.push([...tmp].length);
}
console.log(l.join(" "));